plugins/modules/cloud/alicloud/ali_instance.py

```diff
@@ -807,9 +798,6 @@ def main():
     )
     module = AnsibleModule(argument_spec=argument_spec)
 
-    if HAS_FOOTMARK is False:
```

Comment:
This module and the next one should have a changelog entry. (The other ones don't need one IMO.)

Reply:
@felixfontein I have taken a closer look and adjusted. Please review.